src/libs/routing/routes/sim.tsx

```diff
@@ -0,0 +1,90 @@
+import { Route } from '@tanstack/react-router';
+import { SimulatorProvider } from 'libs/d3';
+import { rootRoute } from 'libs/routing/routes/root';
+import { SimulatorPage } from 'pages/simulator';
+import { SimulatorResultPage } from 'pages/simulator/result';
+
+export const simulatorPage = new Route({
+  getParentRoute: () => rootRoute,
+  path: '/simulator',
+  component: SimulatorPage,
+});
+
+export interface SimulatorSearch {
+  start: string;
+  end: string;
+  baseToken: string;
+  baseBudget: string;
+  quoteToken: string;
+  quoteBudget: string;
+  sellMax: string;
+  sellMin: string;
+  buyMax: string;
+  buyMin: string;
+}
+
+export const simulatorResultPage = new Route({
```

Comment:
We could improve the naming.
SimulatorResultPage - the page component
SimulatorResultPageRoute - the wrapping route